index.js

```diff
@@ -70,5 +76,52 @@ function toggleColor(element) {
 
 /***** Deliverable 2 *****/
 
+const likeBtn = document.querySelector(".like-button")
+const likes = document.querySelector(".likes")
+
+
+function increaseLikes() {
+  const newLikes = parseInt(likes.textContent) + 1
+  likes.textContent = `${newLikes} Likes`
+}
+
+
+likeBtn.addEventListener("click", increaseLikes)
+
 
 /***** Deliverable 3 *****/
+
+// create variable for form, link and description
+const form = document.querySelector("#new-goal-form")
+
+
+
+// const goalId = player.goals.length + 1
+// haha i thought i was gonna be all clever with player.goals.length + 1 as the id -- but the input data doesn't persist. Thus all my new links would have a goal id of 3 lolol. back to the drawing board!
+
+const ul = document.querySelector("ul")
+const goalId = ul.children.length +1
+// boo this didn't work either. All of my new data-ids are 3.... why is that?
```

Comment:
Another possible way of going about this:

```javascript
const newGoalId = player.goals[length - 1].id + 1
```

`player.goals[length - 1]` gets the last goal in the array of goals. Then`.id` to get the id of that one goal. Although, you're still right that it wouldn't persist, so they'd all have a goal id of 3. 😅 But once we're able to persist data, grabbing the data through the player object will be the way to go!